### DISCUSSION TITLE
Watch list syncing but not requesting

### DISCUSSION THREAD

**Discussion by BurningTheSky** [sct/overseerr#3880] [Help]
I've had Overseerr for a while and requesting via Plex watchlists used to work well but recently I notice that watchlist items weren't downloading. I've checked that the config to sonarr and radarr are working, they test fine. 

I don't see any errors in the logs relating to the watchlist but I do see a debug entry:
```
{
 "errorMessage": "Auto-request for this media and user already exists.",
 "mediaTitle": "Title",
 "userId": 1
}
```

Any suggestions on how to resolve?

**Comment by mikedodge04** [comment]
I might be seeing the same issue. Do you see the watch list for userid 1 in the UI?

**Comment by chris5a** [comment]
I just noticed this was no longer working for me too. I don't use it much but remember it working in the past.
<img width="2397" height="962" alt="Plex watchlist not working" src="https://github.com/user-attachments/assets/7f3eed4b-a724-4ebb-a575-39cd9d960bc0" />
